### ISSUE TITLE
Populate standard audit log field for justification

### ISSUE THREAD

**Issue by yolocs** [abcxyz/lumberjack#266]
The field being https://github.com/googleapis/googleapis/blob/master/google/rpc/context/attribute_context.proto#L199-L201

`log.RequestMetadata.RequestAttributes.Reason`

We currently add the justification token to the log metadata. But we should consider also populating the standard `reason` field with the text justification.

cc: @raserva

**Comment by raserva** [comment]
I agree that it could be beneficial to copy the "explanation" field from the justificaiton token to the reason field.

**Comment by yolocs** [comment]
Additional idea: Justification can now be auto populated with gRPC interceptor. But when a user doesn't use the interceptor (e.g. not a gRPC service), the parsing/populating of the justification becomes harder. We can add a lib that 1) verify/parse a given justification token, 2) populate a audit log request with the justification

**Comment by yolocs** [comment]
This becomes more important as some downstream component expecting the justification in `log.RequestMetadata.RequestAttributes.Reason`.
We can json encode the `justs` claim in the JVS token and put that in this field.

**Comment by capri-xiyue** [comment]
golang part https://github.com/abcxyz/lumberjack/pull/293 completed
